mlflow/spark/__init__.py

```diff
@@ -663,7 +716,60 @@ def save_model(
         sample_input: A sample input that is used to add the MLeap flavor to the model.
             This must be a PySpark DataFrame that the model can evaluate. If
             ``sample_input`` is ``None``, the MLeap flavor is not added.
-        signature: {{ signature }}
+        signature: A Model Signature object that describes the input and output Schema of the
```

Comment:
Can we simply add a link to the `log_model` doc to avoid repeating same description?